src/module.c

```diff
@@ -9678,6 +9678,12 @@ RedisModuleString *RM_GetModuleUserACLString(RedisModuleUser *user) {
  * The returned string must be released with RedisModule_FreeString() or by
  * enabling automatic memory management. */
 RedisModuleString *RM_GetCurrentUserName(RedisModuleCtx *ctx) {
+    // Sometimes, the user isn't passed along the call stack or isn't
+    // even set, so we need to check for the members to avoid crashes.
```

Comment:
```suggestion
    /* Sometimes, the user isn't passed along the call stack or isn't
     * even set, so we need to check for the members to avoid crashes. */
```

Reply:
Done.